### ISSUE TITLE
2025-02-19, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#88]
# Today's Arxiv Papers

(1) [Multimodal Mamba: Decoder-only Multimodal State Space Model via Quadratic to Linear Distillation](http://arxiv.org/abs/2502.13145v1), 2025-02-18

(2) [Re-Align: Aligning Vision Language Models via Retrieval-Augmented Direct Preference Optimization](http://arxiv.org/abs/2502.13146v1), 2025-02-18

(3) [UniGuardian: A Unified Defense for Detecting Prompt Injection, Backdoor Attacks and Adversarial Attacks in Large Language Models](http://arxiv.org/abs/2502.13141v1), 2025-02-18

(4) [AIDE: AI-Driven Exploration in the Space of Code](http://arxiv.org/abs/2502.13138v1), 2025-02-18

(5) [Theorem Prover as a Judge for Synthetic Data Generation](http://arxiv.org/abs/2502.13137v1), 2025-02-18

(6) [Learning to Defer for Causal Discovery with Imperfect Experts](http://arxiv.org/abs/2502.13132v1), 2025-02-18

(7) [Rethinking Diverse Human Preference Learning through Principal Component Analysis](http://arxiv.org/abs/2502.13131v1), 2025-02-18

(8) [Facilitating Long Context Understanding via Supervised Chain-of-Thought Reasoning](http://arxiv.org/abs/2502.13127v1), 2025-02-18

(9) [RuozhiBench: Evaluating LLMs with Logical Fallacies and Misleading Premises](http://arxiv.org/abs/2502.13125v1), 2025-02-18

(10) [Adapting Psycholinguistic Research for LLMs: Gender-inclusive Language in a Coreference Context](http://arxiv.org/abs/2502.13120v1), 2025-02-18

(11) [STEER-ME: Assessing the Microeconomic Reasoning of Large Language Models](http://arxiv.org/abs/2502.13119v1), 2025-02-18

(12) [Performance Evaluation of Large Language Models in Statistical Programming](http://arxiv.org/abs/2502.13117v1), 2025-02-18

(13) [MatterChat: A Multi-Modal LLM for Material Science](http://arxiv.org/abs/2502.13107v1), 2025-02-18

(14) [Understanding and Rectifying Safety Perception Distortion in VLMs](http://arxiv.org/abs/2502.13095v1), 2025-02-18

(15) [Text2World: Benchmarking Large Language Models for Symbolic World Model Generation](http://arxiv.org/abs/2502.13092v1), 2025-02-18

(16) [Interactive Agents to Overcome Ambiguity in Software Engineering](http://arxiv.org/abs/2502.13069v1), 2025-02-18

(17) [SimpleVQA: Multimodal Factuality Evaluation for Multimodal Large Language Models](http://arxiv.org/abs/2502.13059v1), 2025-02-18

(18) [LAMD: Context-driven Android Malware Detection and Classification with LLMs](http://arxiv.org/abs/2502.13055v1), 2025-02-18

(19) [AEIA-MN: Evaluating the Robustness of Multimodal LLM-Powered Mobile Agents Against Active Environmental Injection Attacks](http://arxiv.org/abs/2502.13053v1), 2025-02-18

(20) [Do we still need Human Annotators? Prompting Large Language Models for Aspect Sentiment Quad Prediction](http://arxiv.org/abs/2502.13044v1), 2025-02-18

(21) [HPSS: Heuristic Prompting Strategy Search for LLM Evaluators](http://arxiv.org/abs/2502.13031v1), 2025-02-18

(22) [Agentic Deep Graph Reasoning Yields Self-Organizing Knowledge Networks](http://arxiv.org/abs/2502.13025v1), 2025-02-18

(23) [Oreo: A Plug-in Context Reconstructor to Enhance Retrieval-Augmented Generation](http://arxiv.org/abs/2502.13019v1), 2025-02-18

(24) [LLM-Powered Proactive Data Systems](http://arxiv.org/abs/2502.13016v1), 2025-02-18

(25) [Towards a Design Guideline for RPA Evaluation: A Survey of Large Language Model-Based Role-Playing Agents](http://arxiv.org/abs/2502.13012v1), 2025-02-18

(26) [Adaptive Knowledge Graphs Enhance Medical Question Answering: Bridging the Gap Between LLMs and Evolving Medical Knowledge](http://arxiv.org/abs/2502.13010v1), 2025-02-18

(27) [You need to MIMIC to get FAME: Solving Meeting Transcript Scarcity with a Multi-Agent Conversations](http://arxiv.org/abs/2502.13001v1), 2025-02-18

(28) [Personalized Top-k Set Queries Over Predicted Scores](http://arxiv.org/abs/2502.12998v1), 2025-02-18

(29) [Beyond Profile: From Surface-Level Facts to Deep Persona Simulation in LLMs](http://arxiv.org/abs/2502.12988v1), 2025-02-18

(30) [Sailor2: Sailing in South-East Asia with Inclusive Multilingual LLMs](http://arxiv.org/abs/2502.12982v1), 2025-02-18

(31) [Learning More Effective Representations for Dense Retrieval through Deliberate Thinking Before Search](http://arxiv.org/abs/2502.12974v1), 2025-02-18

(32) [Reasoning-to-Defend: Safety-Aware Reasoning Can Defend Large Language Models from Jailbreaking](http://arxiv.org/abs/2502.12970v1), 2025-02-18

(33) [Trust Me, I'm Wrong: High-Certainty Hallucinations in LLMs](http://arxiv.org/abs/2502.12964v1), 2025-02-18

(34) [Infinite Retrieval: Attention Enhanced LLMs in Long-Context Processing](http://arxiv.org/abs/2502.12962v1), 2025-02-18

(35) [Adaptive Tool Use in Large Language Models with Meta-Cognition Trigger](http://arxiv.org/abs/2502.12961v1), 2025-02-18

(36) [Fake It Till You Make It: Using Synthetic Data and Domain Knowledge for Improved Text-Based Learning for LGE Detection](http://arxiv.org/abs/2502.12948v1), 2025-02-18

(37) [Every Expert Matters: Towards Effective Knowledge Distillation for Mixture-of-Experts Language Models](http://arxiv.org/abs/2502.12947v1), 2025-02-18

(38) [LLMPopcorn: An Empirical Study of LLMs as Assistants for Popular Micro-video Generation](http://arxiv.org/abs/2502.12945v1), 2025-02-18

(39) [Synthetic Data Generation for Culturally Nuanced Commonsense Reasoning in Low-Resource Languages](http://arxiv.org/abs/2502.12932v1), 2025-02-18

(40) [Flow-of-Options: Diversified and Improved LLM Reasoning by Thinking Through Options](http://arxiv.org/abs/2502.12929v1), 2025-02-18

(41) [Finedeep: Mitigating Sparse Activation in Dense LLMs via Multi-Layer Fine-Grained Experts](http://arxiv.org/abs/2502.12928v1), 2025-02-18

(42) [SEFL: Harnessing Large Language Model Agents to Improve Educational Feedback Systems](http://arxiv.org/abs/2502.12927v1), 2025-02-18

(43) [Towards more Contextual Agents: An extractor-Generator Optimization Framework](http://arxiv.org/abs/2502.12926v1), 2025-02-18

(44) [Conditioning LLMs to Generate Code-Switched Text: A Methodology Grounded in Naturally Occurring Data](http://arxiv.org/abs/2502.12924v1), 2025-02-18

(45) [On-Device LLMs for Home Assistant: Dual Role in Intent Detection and Response Generation](http://arxiv.org/abs/2502.12923v1), 2025-02-18

(46) [Q-STRUM Debate: Query-Driven Contrastive Summarization for Recommendation Comparison](http://arxiv.org/abs/2502.12921v1), 2025-02-18

(47) [Query Rewriting via LLMs](http://arxiv.org/abs/2502.12918v1), 2025-02-18

(48) [GSQ-Tuning: Group-Shared Exponents Integer in Fully Quantized Training for LLMs On-Device Fine-tuning](http://arxiv.org/abs/2502.12913v1), 2025-02-18

(49) [Knapsack Optimization-based Schema Linking for LLM-based Text-to-SQL Generation](http://arxiv.org/abs/2502.12911v1), 2025-02-18

(50) [Fraud-R1 : A Multi-Round Benchmark for Assessing the Robustness of LLM Against Augmented Fraud and Phishing Inducements](http://arxiv.org/abs/2502.12904v1), 2025-02-18

(51) [Soundwave: Less is More for Speech-Text Alignment in LLMs](http://arxiv.org/abs/2502.12900v1), 2025-02-18

(52) [None of the Others: a General Technique to Distinguish Reasoning from Memorization in Multiple-Choice LLM Evaluation Benchmarks](http://arxiv.org/abs/2502.12896v1), 2025-02-18

(53) [Multilingual European Language Models: Benchmarking Approaches and Challenges](http://arxiv.org/abs/2502.12895v1), 2025-02-18

(54) [Are Multilingual Language Models an Off-ramp for Under-resourced Languages? Will we arrive at Digital Language Equality in Europe in 2030?](http://arxiv.org/abs/2502.12886v1), 2025-02-18

(55) [How desirable is alignment between LLMs and linguistically diverse human users?](http://arxiv.org/abs/2502.12884v1), 2025-02-18

(56) [Continuous Learning Conversational AI: A Personalized Agent Framework via A2C Reinforcement Learning](http://arxiv.org/abs/2502.12876v1), 2025-02-18

(57) [RobotIQ: Empowering Mobile Robots with Human-Level Planning for Real-World Execution](http://arxiv.org/abs/2502.12862v1), 2025-02-18

(58) [PAFT: Prompt-Agnostic Fine-Tuning](http://arxiv.org/abs/2502.12859v1), 2025-02-18

(59) [Rejected Dialects: Biases Against African American Language in Reward Models](http://arxiv.org/abs/2502.12858v1), 2025-02-18

(60) [S$^2$R: Teaching LLMs to Self-verify and Self-correct via Reinforcement Learning](http://arxiv.org/abs/2502.12853v1), 2025-02-18

(61) [MeMo: Towards Language Models with Associative Memory Mechanisms](http://arxiv.org/abs/2502.12851v1), 2025-02-18

(62) [MOLLM: Multi-Objective Large Language Model for Molecular Design -- Optimizing with Experts](http://arxiv.org/abs/2502.12845v1), 2025-02-18

(63) [Towards Adaptive Feedback with AI: Comparing the Feedback Quality of LLMs and Teachers on Experimentation Protocols](http://arxiv.org/abs/2502.12842v1), 2025-02-18

(64) [Towards Equitable AI: Detecting Bias in Using Large Language Models for Marketing](http://arxiv.org/abs/2502.12838v1), 2025-02-18

(65) [An LLM-Powered Agent for Physiological Data Analysis: A Case Study on PPG-based Heart Rate Estimation](http://arxiv.org/abs/2502.12836v1), 2025-02-18

(66) [KazMMLU: Evaluating Language Models on Kazakh, Russian, and Regional Knowledge of Kazakhstan](http://arxiv.org/abs/2502.12829v1), 2025-02-18

(67) [Reasoning and the Trusting Behavior of DeepSeek and GPT: An Experiment Revealing Hidden Fault Lines in Large Language Models](http://arxiv.org/abs/2502.12825v1), 2025-02-18

(68) [Pitfalls of Scale: Investigating the Inverse Task of Redefinition in Large Language Models](http://arxiv.org/abs/2502.12821v1), 2025-02-18

(69) [Simulating User Diversity in Task-Oriented Dialogue Systems using Large Language Models](http://arxiv.org/abs/2502.12813v1), 2025-02-18

(70) [Towards Text-Image Interleaved Retrieval](http://arxiv.org/abs/2502.12799v1), 2025-02-18

(71) [Commonsense Reasoning in Arab Culture](http://arxiv.org/abs/2502.12788v1), 2025-02-18

(72) [How Much Do LLMs Hallucinate across Languages? On Multilingual Estimation of LLM Hallucination in the Wild](http://arxiv.org/abs/2502.12769v1), 2025-02-18

(73) [R2-KG: General-Purpose Dual-Agent Framework for Reliable Reasoning on Knowledge Graphs](http://arxiv.org/abs/2502.12767v1), 2025-02-18

(74) [Efficient Machine Translation Corpus Generation: Integrating Human-in-the-Loop Post-Editing with Large Language Models](http://arxiv.org/abs/2502.12755v1), 2025-02-18

(75) [Self-Enhanced Reasoning Training: Activating Latent Reasoning in Small Models for Enhanced Reasoning Distillation](http://arxiv.org/abs/2502.12744v1), 2025-02-18

(76) ["I know myself better, but not really greatly": Using LLMs to Detect and Explain LLM-Generated Texts](http://arxiv.org/abs/2502.12743v1), 2025-02-18

(77) [Circuit Representation Learning with Masked Gate Modeling and Verilog-AIG Alignment](http://arxiv.org/abs/2502.12732v1), 2025-02-18

(78) [TREND: A Whitespace Replacement Information Hiding Method](http://arxiv.org/abs/2502.12710v1), 2025-02-18

(79) [Multi-Novelty: Improve the Diversity and Novelty of Contents Generated by Large Language Models via inference-time Multi-Views Brainstorming](http://arxiv.org/abs/2502.12700v1), 2025-02-18

(80) [Multi-Step Alignment as Markov Games: An Optimistic Online Gradient Descent Approach with Convergence Guarantees](http://arxiv.org/abs/2502.12678v1), 2025-02-18

(81) [Baichuan-M1: Pushing the Medical Capability of Large Language Models](http://arxiv.org/abs/2502.12671v1), 2025-02-18

(82) [Perovskite-LLM: Knowledge-Enhanced Large Language Models for Perovskite Solar Cell Research](http://arxiv.org/abs/2502.12669v1), 2025-02-18

(83) [Evaluation of Best-of-N Sampling Strategies for Language Model Alignment](http://arxiv.org/abs/2502.12668v1), 2025-02-18

(84) [A$^2$ATS: Retrieval-Based KV Cache Reduction via Windowed Rotary Position Embedding and Query-Aware Vector Quantization](http://arxiv.org/abs/2502.12665v1), 2025-02-18

(85) [Demystifying Multilingual Chain-of-Thought in Process Reward Modeling](http://arxiv.org/abs/2502.12663v1), 2025-02-18

(86) [The Hidden Risks of Large Reasoning Models: A Safety Assessment of R1](http://arxiv.org/abs/2502.12659v1), 2025-02-18

(87) [R.R.: Unveiling LLM Training Privacy through Recollection and Ranking](http://arxiv.org/abs/2502.12658v1), 2025-02-18

(88) [Corrupted but Not Broken: Rethinking the Impact of Corrupted Data in Visual Instruction Tuning](http://arxiv.org/abs/2502.12635v1), 2025-02-18

(89) [\textit{One Size doesn't Fit All}: A Personalized Conversational Tutoring Agent for Mathematics Instruction](http://arxiv.org/abs/2502.12633v1), 2025-02-18

(90) [Automating Prompt Leakage Attacks on Large Language Models Using Agentic Approach](http://arxiv.org/abs/2502.12630v1), 2025-02-18

(91) [DeepResonance: Enhancing Multimodal Music Understanding via Music-centric Multi-way Instruction Tuning](http://arxiv.org/abs/2502.12623v1), 2025-02-18

(92) [Improving Chain-of-Thought Reasoning via Quasi-Symbolic Abstractions](http://arxiv.org/abs/2502.12616v1), 2025-02-18

(93) [Who Writes What: Unveiling the Impact of Author Roles on AI-generated Text Detection](http://arxiv.org/abs/2502.12611v1), 2025-02-18

(94) [COPU: Conformal Prediction for Uncertainty Quantification in Natural Language Generation](http://arxiv.org/abs/2502.12601v1), 2025-02-18

(95) [Bring Your Own Knowledge: A Survey of Methods for LLM Knowledge Expansion](http://arxiv.org/abs/2502.12598v1), 2025-02-18

(96) [PASER: Post-Training Data Selection for Efficient Pruned Large Language Model Recovery](http://arxiv.org/abs/2502.12594v1), 2025-02-18

(97) [RM-PoT: Reformulating Mathematical Problems and Solving via Program of Thoughts](http://arxiv.org/abs/2502.12589v1), 2025-02-18

(98) [G-Refer: Graph Retrieval-Augmented Large Language Model for Explainable Recommendation](http://arxiv.org/abs/2502.12586v1), 2025-02-18

(99) [LongFaith: Enhancing Long-Context Reasoning in LLMs with Faithful Synthetic Data](http://arxiv.org/abs/2502.12583v1), 2025-02-18

(100) [DemonAgent: Dynamically Encrypted Multi-Backdoor Implantation Attack on LLM-based Agent](http://arxiv.org/abs/2502.12575v1), 2025-02-18
